k8s/esp_v2_echo_gke.yaml

```diff
@@ -0,0 +1,55 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: esp-echo
+spec:
+  ports:
+  - port: 80
+    targetPort: 8080
+    protocol: TCP
+    name: http
+  selector:
+    app: esp-echo
+  type: LoadBalancer
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: esp-echo
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: esp-echo
+    spec:
+      containers:
+      - name: esp
+        image: gcr.io/endpoints-release/endpoints-runtime:2
+        args: [
+          "--listener_port", "8080",
+          "--backend", "127.0.0.1:8081",
+          "--service", "SERVICE_NAME",
+          "--rollout_strategy", "managed",
+        ]
+        ports:
+          - containerPort: 8080
+      - name: echo
+        image: gcr.io/endpoints-release/echo:latest
```

Comment:
So is this for a specific language? Or will we use this yaml file for all languages?

Reply:
No,  gke yaml files added here are not language specific.